### DISCUSSION TITLE
Deadlineem deserves more attention and respect from people

### DISCUSSION THREAD

**Discussion by infinity-stack** [Deadlineem/Extras-Addon-for-YimMenu#127] [General]
he deserves everything because he fixes things that his community has bugs with and I'm not saying that cause I'm dick riding but I'm saying it cause he is underrated and deserves better

**Comment by Deadlineem** [comment]
Lol thanks i guess.  I am the one who started the project so I try to keep things fixed promptly, but i for sure don't want all the credit, alot of this stuff is from many others in the community.

**Comment by infinity-stack** [comment]
evb love u bro
